packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -40,18 +76,34 @@ export const TagSubforumPage = ({ classes, user }: { classes: ClassesType; user:
     return <Error404 />;
   }
 
-  if ((error && !isMissingDocumentError(error))) {
+  if (error && !isMissingDocumentError(error)) {
     return (
       <SingleColumnSection>
         <Typography variant="body1">{error.message}</Typography>
       </SingleColumnSection>
     );
   }
 
+  const welcomeBoxComponent = (
+    <div className={classes.welcomeBoxPadding}>
+      <div className={classes.welcomeBox}>
+        <ContentStyles contentType="comment">
+          <ContentItemBody
+            dangerouslySetInnerHTML={{ __html: tag.subforumWelcomeText?.html || "" }}
+            description={`${tag.name} subforum`}
+          />
+        </ContentStyles>
+      </div>
+    </div>
+  );
+
   return (
     <div className={classes.root}>
+      <SingleColumnSection className={classNames(classes.columnSection, classes.stickToBottom)}>
+        {welcomeBoxComponent}
+      </SingleColumnSection>
       <SingleColumnSection className={classes.columnSection}>
```

Comment:
Currently, this middle column can get squished by the surrounding columns - I believe adding `flex: "none"` will fix it

<img width="700" alt="Screen Shot 2022-09-12 at 2 08 24 PM" src="https://user-images.githubusercontent.com/9057804/189728021-c1ee22b6-d39b-47c0-b30e-16d5a1b33161.png">

Reply:
Ah thanks, this wasn't happening on the forecasting subforum I was testing on, presumably because there were comments long enough to make it fill the full width